catalog/glue/glue.go

```diff
@@ -228,7 +224,7 @@ func (c *Catalog) LoadTable(ctx context.Context, identifier table.Identifier, pr
 		ctx,
 		identifier,
 		location,
-		io.LoadFSFunc(props, location),
+		io.LoadFSFunc(nil, location),
```

Comment:
if we're not passing along properties that are passed in, then we should at least be passing along properties from the catalog. the properties passed in to `io.LoadFS` are per-table configuration properties that could matter such as controlling properties involving writes, manifest merging, and so on. I'm not particularly comfortable with removing this entirely unless there's some other way for a user to provide these properties that could affect reads